latex/ledger-spec.tex

```diff
@@ -186,16 +203,77 @@ \section{Serialization}
     serialization}
 \end{todo}
 
+\section{Delegation}
+\label{sec:delegation}
+\input{delegation.tex}
+
 \section{UTxO}
 \label{sec:state-trans-utxo-1}
 
+In order to define the \textit{preservation of value} conditon,
+we define the calculations for deposits and refunds in
+Figure~\ref{fig:defs:deposits}.
+
 The transition rules for unspent outputs are presented in
 Figure~\ref{fig:rules:utxo}. The states of the UTxO transition system,
 along with their types are defined in Figure~\ref{fig:defs:utxo}.
 Functions on these types are defined in Figure~\ref{fig:derived-defs:utxo}.
 
+
+
+\begin{figure*}
+  \emph{Abstract types}
+  \begin{equation*}
+    \begin{array}{r@{~\in~}lr}
+      pc & \PrtclConsts & \text{protocol constants}
+    \end{array}
+  \end{equation*}
+  %
+  \emph{Abstract Functions}
+  \begin{equation*}
+    \begin{array}{r@{~\in~}lr}
+      \fun{d} & \PrtclConsts \to \DCert \to \Coin
+        & \text{deposit amount of a certificate}\\
+
+      \fun{decay} & \PrtclConsts \to \mathbb{N}\times\mathbb{Q}^{+}
+        & \text{decay constants}\\
+
+      \fun{certs} & \Tx \to \powerset{\DCert}
+        & \text{certificates}\\
+      %
+      \fun{ttl} & \Tx \to \Slot
+        & \text{time to live}\\
+    \end{array}
+  \end{equation*}
+  \caption{Definitions used in Deposits}
+  \label{fig:defs:deposits}
+\end{figure*}
+
+\begin{figure}
+  \begin{align*}
+      & \fun{deposits} \in \PrtclConsts \to \Tx \to \Coin
+      & \text{total deposits for transaction} \\
+      & \fun{deposits}~{pc}~{tx} = \sum\limits_{c \in \fun{certs}~tx} (\fun{d}~pc~c)
+      \nextdef
+      & \fun{refund} \in \PrtclConsts \to \Slot \to \DCert \to \Coin
+      & \text{total refund for a certificate} \\
+      & \fun{refund}~{pc}~{slot}~{c} =
+      \floor*{
+        \left(\fun{d}~pc~c\right) \cdot
+      \left(d_{\min}+(1-d_{\min})\cdot e^{-\lambda\cdot\var{slot}}\right)}\\
+      & \where \fun{decay}~pc = d_{\min},~\lambda
+      \nextdef
+      & \fun{refunds} \in \PrtclConsts \to \Tx \to \Coin
+      & \text{total refunds for transaction} \\
+      & \refunds{pc}{tx} = \sum\limits_{c \in \fun{certs}~tx} \fun{refund}~pc~(\ttl{tx})~c
```

Comment:
I just realized that this is not correct, I'll fix this shortly.  First of all, only certificates that release resources get refunds.  Secondly, the slot parameter given to `refund` should not be `ttl tx`, but rather `(ttl tx) - X`, where X is the time that the corresponding creation certificate was made. We need to be storing these creation slots.